package.json

```diff
@@ -1,5 +1,5 @@
 {
   "dependencies": {
-    "@okta/openapi": "2.6.0"
+    "@okta/openapi": "2.7.1"
```

Comment:
Can we wait for 2.7.2 and update it directly to that version?

Reply:
Sure, it should be 2.7.2
But for now, this version was not published yet so I am avoiding "not found error"

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @okta/openapi@2.7.2.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

Reply:
updated now